src/clj/game/core.clj

```diff
@@ -63,7 +63,12 @@
     [game.macros]
     [potemkin :refer [import-vars]]))
 
-(import-vars
+(defmacro expose-vars
+  "Surrounds a import-vars call with an anonymous function to get arround the 64kb limit on the method size in the JVM" 
+  [symbols]
+  `((fn ~[] (import-vars ~symbols))))
```

Comment:
I don't think you need the unquote in front of the arg vector, but maybe you do.